docs/walkthrough/integrate-with-jina.md

```diff
@@ -0,0 +1,72 @@
+(integrate-with-jina)=
+# Integrate with Jina
+
+Once fine-tuning is finished, it's time to actually use the model. 
+Finetuner, being part of the Jina ecosystem, provides a convenient way to use tuned models via [Jina Executors](https://docs.jina.ai/fundamentals/executor/).
+
+We've created the [`FinetunerExecutor`](https://hub.jina.ai/executor/13dzxycc) which can be added in a [Jina Flow](https://docs.jina.ai/fundamentals/flow/) and load any tuned model. 
+
+````{tab} via Docker image (recommended)
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub+docker://FinetunerExecutor', uses_with=...)
+```
+````
+````{tab} via source code
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub://FinetunerExecutor', uses_with=...)
+```
+````
```

Comment:
Can we add a minimal example config to the `uses_with=`? To show people at a glance which keys are required, without them having to look at the list below.